--- krecipes-0.9-orig/configure	2005-11-29 16:40:35.000000000 -0600
+++ krecipes-0.9/configure	2005-12-03 22:18:40.000000000 -0600
@@ -27972,7 +27972,7 @@
 
 if test "$ac_x_includes" = NO; then
   # Guess where to find include files, by looking for this one X11 .h file.
-  test -z "$x_direct_test_include" && x_direct_test_include=X11/Intrinsic.h
+  test -z "$x_direct_test_include" && x_direct_test_include=X11/Xlib.h
 
   # First, try using that file with no special directory specified.
 cat >conftest.$ac_ext <<_ACEOF
@@ -28057,7 +28057,7 @@
 if test "$ac_x_libraries" = NO; then
   # Check for the libraries.
 
-  test -z "$x_direct_test_library" && x_direct_test_library=Xt
+  test -z "$x_direct_test_library" && x_direct_test_library=X11
   test -z "$x_direct_test_function" && x_direct_test_function=XtMalloc
 
   # See if we find them without any special options.
