diff -up krecipes-0.9.1/krecipes/src/importers/kreimporter.h.gcc43 krecipes-0.9.1/krecipes/src/importers/kreimporter.h
--- krecipes-0.9.1/krecipes/src/importers/kreimporter.h.gcc43	2005-12-03 15:33:56.000000000 -0600
+++ krecipes-0.9.1/krecipes/src/importers/kreimporter.h	2008-03-31 11:36:53.000000000 -0500
@@ -46,7 +46,7 @@ private:
 	void readCategoryStructure( const QDomNodeList& l, CategoryTree *tree );
 	void readDescription( const QDomNodeList& l, Recipe* );
 	void readIngredients( const QDomNodeList& l, Recipe*, const QString &header = QString::null );
-	void readAmount( const QDomElement& amount, double &amount, double &amount_offset );
+	void readAmount( const QDomElement& amount, double &amount2, double &amount_offset );
 	void readRatings( const QDomNodeList&, Recipe * );
 	void readCriterion( const QDomNodeList&, RatingCriteriaList &r );
 };
diff -up krecipes-0.9.1/krecipes/src/importers/recipemlimporter.h.gcc43 krecipes-0.9.1/krecipes/src/importers/recipemlimporter.h
--- krecipes-0.9.1/krecipes/src/importers/recipemlimporter.h.gcc43	2005-12-03 15:33:56.000000000 -0600
+++ krecipes-0.9.1/krecipes/src/importers/recipemlimporter.h	2008-03-31 11:36:23.000000000 -0500
@@ -45,7 +45,7 @@ private:
 	void readRecipemlRecipe( const QDomElement& recipe );
 	void readRecipemlPreptime( const QDomElement &preptime );
 	void readRecipemlQty( const QDomElement &qty, Ingredient &ing );
-	void readRecipemlRange( const QDomElement& range, double &range, double &range_offset );
+	void readRecipemlRange( const QDomElement& range, double &range2, double &range_offset );
 
 	Recipe recipe;
 };
